### ISSUE TITLE
19+1 setting

### ISSUE THREAD

**Issue by aooating** [akshitac8/OW-DETR#19]
In this setting, can you send me the trained model? Because there is a gap between my results and the original.

**Comment by Wang0203** [comment]
Hello! @aooating Is there a gap between your results and the original in other settings(10+10 setting or 15+5setting)?

**Comment by Wang0203** [comment]
Hello! @aooating Is there a gap between your results and the original in other settings(10+10 setting or 15+5setting)?

**Comment by aooating** [comment]
> Hello! @aooating Is there a gap between your results and the original in other settings(10+10 setting or 15+5setting)?

Yes. I didn't train the 10+10 setting. In the 19+1 and 15+5 Settings, there is a gap.

**Comment by Wang0203** [comment]
> 
How do you train it? What's your details of the data?
